### PR TITLE
Contains tests

### PR DESCRIPTION
**Pull request by EmanuP**
Cobol.groovy script has been modified to build cobol testcases independently of the flag "runzTests".

The code performance was improved with respect to the previous pull request.

### CODE REVIEW

File: languages/Cobol.groovy [hunk 1]
```diff
@@ -339,3 +339,18 @@ def getRepositoryClient() {
...
+boolean buildListContainsTests(List<String> buildList) {
+	boolean testFound = false
+	// iterate through build list
+	buildList.each { buildFile ->
+	
+		// Check if this is a testcase
+		isZUnitTestCase = (props.getFileProperty('cobol_testcase', buildFile).equals('true')) ? true : false
+		
+		if (isZUnitTestCase) {
+			testFound = true
```
Comment (dennis-behm) [inline languages/Cobol.groovy:352]: We can directly `return true` if the first test is found, instead of integrating through the entire buildList.

**Review by dennis-behm — CHANGES_REQUESTED** [review]
Hi @EmanuP, thanks for the PR and apologies for the very late response. 
I like to idea to just create the datasets in case there is a test involved.
Can you please have a look to the two comments?

File: languages/Cobol.groovy (hunk 1 above)
Comment (EmanuP) [reply languages/Cobol.groovy:352]: Good idea, this will improve performance.

**Comment by EmanuP** [comment]
@dennis-behm Does the last commit reflect your idea?

File: languages/Cobol.groovy [hunk 2]
```diff
@@ -339,3 +339,17 @@ def getRepositoryClient() {
...
+
+boolean buildListContainsTests(List<String> buildList) {
+	// iterate through build list
+	buildList.each { buildFile ->
+	
+		// Check if this is a testcase
+		isZUnitTestCase = (props.getFileProperty('cobol_testcase', buildFile).equals('true')) ? true : false
+		
+		if (isZUnitTestCase) {
+			return true
```
Comment (dennis-behm) [inline languages/Cobol.groovy:351]: What I understood from the docs, you cannot break/return out from an each{closure} loop, that is why your current method always returns `false` and the datasets are not allocated.

Let's just switch to a `return isZUnitTestCase` at the very end.

**Review by dennis-behm — CHANGES_REQUESTED** [review]
Please see the comment. The current code always returns false.

File: languages/Cobol.groovy (hunk 2 above)
Comment (dennis-behm) [reply languages/Cobol.groovy:351]: @EmanuP 

Can you please replace the method by, I already tested the code. This should be fine:

```
// check build list for test cases
def boolean buildListContainsTests(List<String> buildList) {

	boolean containsZUnitTestCase = buildList.find { buildFile -> props.getFileProperty('cobol_testcase', buildFile).equals('true')}
	
	return containsZUnitTestCase ? true : false
}
```

**Review by dennis-behm — APPROVED** [review]

**Comment by dennis-behm** [comment]
@jbyibm, this fixes #81

**Pull request merged**